assets/js/components/adminbar/AdminBarClicks.js

```diff
@@ -73,17 +76,17 @@ function AdminBarClicks( { WidgetReportZero, WidgetReportError } ) {
 		] )
 	);
 
-	if ( ! hasFinishedResolution ) {
-		return <PreviewBlock width="auto" height="59px" />;
-	}
-
 	if ( error ) {
 		return (
 			<WidgetReportError moduleSlug="search-console" error={ error } />
 		);
 	}
 
-	if ( isZeroReport( searchConsoleData ) ) {
+	if ( ! hasFinishedResolution || isGatheringData === undefined ) {
+		return <PreviewBlock width="auto" height="59px" />;
+	}
```

Comment:
Can you clarify why you moved this check overall? Why can't it remain above the `error` check like it has been so far?

Reply:
@felixarntz I think i know what happened. Basically I searched for `isGatheringData` in the codebase and I came across a file which had the `error` component rendered before the loading one. I thought I had to move it also but turns out it's not the same. So I reverted all the changes I made where I moved the component rendering the error.